### PR TITLE
Added script to create custom policy for KMS Key Metadata Access

### PR DESCRIPTION
**Pull request by ankitsrao**

### CODE REVIEW

File: scripts/aws-additional-policies/updateKmsKeyAccessPolicy.py [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+SYNOPSIS
+    Script to grant KMS Key metadata reader access to CSPM Data Collector IAM Role.
+
+.DESCRIPTION
+    - This script will update the Key Access Policy [ all Keys in the AWS Account ] to grant Key metadata Reader access to the CSPM Data Collector IAM Role
+    - This script does not provide any write permissions on the Key
+
+.NOTES
+
+  Copyright (c) Cloudneeti. All rights reserved.
```
Comment (rahulkhengare) [inline scripts/aws-additional-policies/updateKmsKeyAccessPolicy.py:11]: put zscaler

File: scripts/aws-additional-policies/updateKmsKeyAccessPolicy.py [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+                                        usage='%(prog)s [options]',
+                                        description='Update KMS Key Policy')
+
+    arg_parser.add_argument('--cspmRoleArn',
+                            type=str,
+                            required=True,
+                            help='CSPM Data Collector Role ARN')
+
+    args = arg_parser.parse_args()
+    main(args)
```
Comment (rahulkhengare) [inline scripts/aws-additional-policies/updateKmsKeyAccessPolicy.py:124]: Add one newline

File: scripts/aws-additional-policies/updateKmsKeyAccessPolicy.py (hunk 1 above)
Comment (ankitsrao) [reply scripts/aws-additional-policies/updateKmsKeyAccessPolicy.py:11]: Added required updates.

File: scripts/aws-additional-policies/updateKmsKeyAccessPolicy.py (hunk 2 above)
Comment (ankitsrao) [reply scripts/aws-additional-policies/updateKmsKeyAccessPolicy.py]: Added newline as suggested.

**Review by rahulkhengare — APPROVED** [review]

**Pull request merged**